Hazel/src/Platform/OpenGL/OpenGLShader.cpp

```diff
@@ -73,18 +73,21 @@ namespace Hazel {
 
 		const char* typeToken = "#type";
 		size_t typeTokenLength = strlen(typeToken);
-		size_t pos = source.find(typeToken, 0);
+		size_t pos = source.find(typeToken, 0);                                             //Start of shader type declaration line
```

Comment:
I would suggest to just put the comments next to the line (single space), as due to the tabs, the indentations of the comments is wrong no matter what...

Reply:
Done.